rules/windows/persistence_via_wmi_stdregprov_run_services.toml

```diff
@@ -10,7 +10,7 @@ Identifies use of the Windows Management Instrumentation StdRegProv (registry pr
 registry locations for persistence.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
+index = ["logs-endpoint.events.*"]
```

Comment:
The integration is only parsing DWORD and QWORD registry values, so registry.data.strings will always be null here. Dropping Support